logview/frontend/src/services/adapter.ts

```diff
@@ -0,0 +1,26 @@
+import axios, { AxiosInstance } from 'axios';
```

Comment:
AxiosInstance is not used anywhere, you can remove it

Reply:
fixed